src/main/resources/templates/fragments/userBar.html

```diff
@@ -9,32 +9,25 @@
   <nav class="content">
     <ul class="mobile-hidden" id="navigation">
       <li>
-        <ul class="mobile-hidden"
-            th:if="${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}">
-          <li class="user" id="signed-in-user" th:text="${userModel.getId() + ':'}"></li>
-          <li><a
-              class="piwik-event"
-              data-event-id="Your details - top menu" id="your-details"
-              th:href="@{{chsAccountUrl}/user/account(chsAccountUrl=${@environment.getProperty('account.url')})}">Your
-            details
-          </a></li>
-          <li><a
-              class="piwik-event"
-              data-event-id="Your Applications" id="your-applications"
-              th:href="@{{manageApplicationsUrl}/(manageApplicationsUrl=${@environment.getProperty('manageApplications.url')})}">Your
-            Applications
-          </a></li>
+        <ul th:if="${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}"
+            th:remove="tag">
+          <li class="user" id="signed-in-user" th:text="${userModel.getId()}"></li>
+          <li><a class="govuk-link"
+                 th:href="@{{forum}(forum=${@environment.getProperty('developerforum.url')})}">Developer
+            hub forum</a></li>
           <li><a class="piwik-event" data-event-id="Sign out" href="/signout" id="user-signout">Sign
             out
           </a></li>
         </ul>
       </li>
-      <li th:if="!${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}">
-        <!--  -->
-        <a class="piwik-event govuk-link" data-event-id="Sign In" href="/signin" id="user-signin">Sign
-          in /
-          Register</a>
-      </li>
+      <ul th:if="!${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}"
```

Comment:
Do we need this if statement, shouldn't this be the default unless the first if-statement is fulfilled? Can we look at using if-else, or switch statements if removing this doesn't work for whatever reasonn.

Reply:
this is thymeleaf not java... It doesn't have if else.
they have this odd choose thing? I'll see what I can do but might just ignore you on this.

Reply:
I realise its thymeleaf - there are alternatives I listed above such as using a switch statement  or if-unless (although i'm not to keen on that). Don't see the point in evaluating the statement twice if you're not signed in.

Reply:
Even with a choose (or case statement (as switches are called in thymeleaf. Because its not java.)) we'd still be evaluating it multiple times because its not an if else.
It has to evaluate with the possibility that it could be either, both or neither...

Reply:
https://www.thymeleaf.org/whatsnew20.html#swit
I was mistaken. Switch statements exit as soon as they find a match.